packages/p2p/src/stores/buy-sell-store.js

```diff
@@ -89,6 +89,13 @@ export default class BuySellStore extends BaseStore {
 
     @computed
     get rendered_items() {
+        const filtered_items = this.items.filter(item => {
+            if (this.table_type === 'buy') {
+                return item.type === 'sell';
+            }
+            return item.type === 'buy';
+        });
```

Comment:
why're we filtering here?

Reply:
when you quickly switch between the buy - sell tabs, they the adverts that show are both buy and sell ads, and I decided to filter it within the get rendered_items, so that it would only return either buy or sell ads